src/wolfgame.py

```diff
@@ -2440,7 +2440,9 @@ def stop_game(cli, winner = "", abort = False, additional_winners = None):
 
     if not abort:
         cli.msg(chan, gameend_msg)
-
+        # Message Players in Deadchat letting them know that the game has ended. cyberfawkes
+        mass_privmsg(cli, var.DEADCHAT_PLAYERS, messages["endgame_deadchat"].format(chan))
```

Comment:
I'll fix it right now, I forgot to remove my name. I left it in there for debugging so I didn't have to remember the specific line that line of code was on, I could just search "cyberfawkes".